ui/src/pages/home/home.tsx

```diff
@@ -1,29 +1,25 @@
-import Drawer from '@components/Drawer.tsx';
-import Card from '@components/Card.tsx';
+import Card from '@components/Card';
 import { t } from 'i18next';
+import paths from '@routes/paths';
+import { useNavigate } from 'react-router-dom';
 
 export default function Home() {
+  const navigate = useNavigate();
+
   return (
     <>
-      <Drawer />
-      <div className="flex flex-col gap-10 p-14">
-        <div className="flex flex-col gap-0 leading-none">
-          {t('header.welcome')}
-          <span className="block text-xl font-bold">Jacek</span>
-          {t('header.mainPage')}
-        </div>
-
+      <div className="flex max-h-screen min-h-screen flex-col gap-10">
         <Card
-          className="h-40 rounded-normal bg-white shadow-extra hover:cursor-pointer"
+          className="flex-1 rounded-normal bg-white shadow-extra hover:cursor-pointer"
```

Comment:
Divy powinny mieć po równo, dopóki rysunek nie powiększy diva, image powinien mieć maksymalny height diva, ale to nie działa - robi max height obrazka.